src/PayID/pay-id-error.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Types of errors that originate from PayID.
+ */
+export enum PayIDErrorType {
+  Unimplemented,
+}
+
+/**
+ * Represents errors thrown by PayID components of the Xpring SDK.
+ */
+export default class PayIDError extends Error {
```

Comment:
👍 You're always diligent about defining custom errors, which is something I need to get better at.